src/NServiceBus.WindowsPerformanceCounters/PerformanceCountersExtensions.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace NServiceBus
+{
+    using System;
+    using WindowsPerformanceCounters;
+    using Configuration.AdvanceExtensibility;
+
+    public static class PerformanceCountersExtensions
+    {
+        /// <summary>
+        /// Enables the NServiceBus specific performance counters with a specific EndpointSLA.
+        /// </summary>
+        /// <param name="performanceCounters">The <see cref="PerformanceCounters" /> instance to apply the settings to.</param>
+        public static void EnableSLAPerformanceCounters(this PerformanceCounters performanceCounters)
+        {
+            Guard.AgainstNull(nameof(performanceCounters), performanceCounters);
+            performanceCounters.EndpointConfiguration.EnableFeature<SLAMonitoringFeature>();
+        }
+
+        /// <summary>
+        /// Enables the NServiceBus specific performance counters with a specific EndpointSLA.
+        /// </summary>
+        /// <param name="performanceCounters">The <see cref="PerformanceCounters" /> instance to apply the settings to.</param>
+        /// <param name="sla">The <see cref="TimeSpan" /> to use oa the SLA. Must be greater than <see cref="TimeSpan.Zero" />.</param>
+        public static void EnableSLAPerformanceCounters(this PerformanceCounters performanceCounters, TimeSpan sla)
```

Comment:
Just move this method to the `PerformanceCounters` class? (no need for a ext method)